src/scripts/generate-hounds.test.ts

```diff
@@ -0,0 +1,9 @@
+import { genomeNumberToImageIndex } from "./generate-hounds";
+import { decodeBase62Quadlet } from "./hound-genome";
+
+describe("Map quadlet to asset index", () => {
```

Comment:
What do you think about another test case for invalid base10quadlets, just to show that the function can handle such cases too?

Reply:
done 👍